extensions/amp-video/1.0/base-element.js

```diff
@@ -87,20 +87,24 @@ VideoBaseElement['props'] = {
   'artist': {attr: 'artist'},
   'artwork': {attr: 'artwork'},
   'attribution': {attr: 'attribution'},
-  'autoplay': {attr: 'autoplay', type: 'boolean'},
-  'controls': {attr: 'controls', type: 'boolean'},
+  'autoplay': {attr: 'autoplay', type: 'boolean', media: true},
```

Comment:
I think most of these don't really make sense, except for `dock` and `rotate-to-fullscreen`.

Reply:
Removed the others except `controls` - don't you think it's reasonable to show controls or not based on screen size?

Reply:
Sort of. But I think there's a really complicated interplay usually between `autoplay`, `controls` and `noaudio`. Let's confirm this with @alanorozco . It might be that if we can't do it for one of these, we can't do it for all.

Reply:
I agree with Dima, only `dock` and `r-t-f` make sense to me.

> don't you think it's reasonable to show controls or not based on screen size?

A lack of `controls` implies intents distinct enough that I do *not* think it should depend on size. For example, a video without controls is usually presented more like a gif or a presentation detail.

Reply:
Awesome, thanks for weighing in Alan! Removed.